lib/Auth/Process/COmanageDbClient.php

```diff
@@ -1006,6 +1075,9 @@ private function retrieveCOPersonData(&$state)
         } else {
             $basicInfo = $this->getBasicInfo($orgId);
         }
+        // XXX Add CO Person Id in the state
+        $state['rciamAttributes']['registryUserId'] = $basicInfo['id'];
+
```

Comment:
Add note to the filter documentation in the README:
The filter adds the following attributes to the state information that SimpleSAMLphp stores in association with the request:
```
$state['rciamAttributes`] => array(
    'registryUserId` => // User's COPerson ID 
    'aup' => array( ...
)
```